guides/common/modules/proc_upgrading-a-disconnected-project-server.adoc

```diff
@@ -44,107 +44,135 @@ This prevents the upgrade process from overwriting your DNS and DHCP configurati
 
-.. Edit the `/etc/yum.repos.d/satellite-maintenance.repo` file:
+. Optional: If you made manual edits to DNS or DHCP configuration files and do not want to overwrite the changes, enter the following command:
 +
-[options="nowrap"]
+[options="nowrap" subs="attributes"]
 ----
-# vi /etc/yum.repos.d/satellite-maintenance.repo
+# {foreman-installer} --foreman-proxy-dns-managed=false \
+--foreman-proxy-dhcp-managed=false
 ----
 
-... Change the default `InstallMedia` repository name to `{Project}-Maintenance`:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-[{Project}-Maintenance]
-----
+. In the {ProjectWebUI}, navigate to *Hosts* > *Discovered hosts*.
+If there are discovered hosts available, turn them off and then delete all entries under the *Discovered hosts* page.
+Use the organization setting menu to select each remaining organization and repeat the action as needed. 
+Reboot these hosts after the upgrade has completed.
 
-.. Add the `baseurl` directive:
+. Remove old repositories:
 +
-[options="nowrap"]
+[options="nowrap" subs="attributes"]
 ----
-baseurl=file:///media/sat6/Maintenance/
+# rm /etc/yum.repos.d/*
 ----
 
-ifdef::katello,satellite,orcharhino[]
-. If you use {advisorengine} in your environment, update to the latest images:
-* To update using the {Project} installation ISO image, set up containers on your {ProjectServer}:
+. Identify the location of the copied exported directories from the syncable export on the connected {ProjectServer}.
+. Create the `/etc/yum.repos.d/upgrade.repo` and update the `baseurl` directive accordingly:
 +
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# /media/sat6/containers/setup_containers
+[options="nowrap" subs="+quotes,attributes"]
 ----
-* Alternatively, to update by transferring the latest image from a connected system to a disconnected {ProjectServer}, follow these steps:
-... On the connected system, export the container image:
+[{RepoRHEL9BaseOS}]
+name=Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs)
+metadata_expire=-1
+gpgcheck=0
+cost=500
+baseurl=file:///<export baseos location>/content/dist/rhel9/9/x86_64/baseos/os
+[{RepoRHEL9AppStream}]
+name=Red Hat Enterprise Linux 9 for x86_64 - AppStream (RPMs)
+mediaid=None
+metadata_expire=-1
+gpgcheck=0
+cost=500
+baseurl=file:///<export appstream location>/content/dist/rhel9/9/x86_64/appstream/os
+[{Project}]
+name={Project}
+mediaid=None
+metadata_expire=-1
+gpgcheck=0
+cost=500
+baseurl=file:///<export location>/content/dist/layered/rhel9/x86_64/{project-context}/{ProjectVersion}/os/
+----
++
+. Upgrade {foreman-maintain} to the next version:
 +
-[options="nowrap", subs="+quotes,verbatim,attributes"]
+[options="nowrap" subs="attributes"]
 ----
-# skopeo copy containers-storage:registry.redhat.io/satellite/iop-advisor-engine-rhel9:{ProjectVersion} \
-oci-archive:/tmp/iop-advisor-engine.tar
+# {foreman-maintain} self-upgrade --maintenance-repo-label={Project}
 ----
```

Comment:
This looks like a mix of old steps from previous versions that sneaked in here again and lines that you're also adding below. I believe you should delete all of this.